spec/AuthenticationAdapters.spec.js

```diff
@@ -1139,7 +1139,7 @@ describe('apple signin auth adapter', () => {
 
   it('should throw error with missing id_token', async () => {
     try {
-      await apple.validateAuthData({}, { client_id: 'secret' });
+      await apple.validateAuthData({}, { clientIDs: ['secret'] });
```

Comment:
This should be backwards compatible. Accept both string and array.

Write a test for string, one and multiple clientId.

Reply:
Made it backwards compatible, & created tests that test for both string & array